iac-extensions/arm/src/main/java/org/sonar/iac/arm/checks/CertificateBasedAuthenticationCheck.java

```diff
@@ -50,11 +49,13 @@ public class CertificateBasedAuthenticationCheck extends AbstractArmResourceChec
   @Override
   protected void registerResourceConsumer() {
     register("Microsoft.ApiManagement/service/gateways/hostnameConfigurations", resource -> resource.property("negotiateClientCertificate")
-      .reportIf(CertificateBasedAuthenticationCheck::isBooleanFalse, DISABLED_CERTIFICATE_MESSAGE)
+      .reportIf(isBooleanFalse(), DISABLED_CERTIFICATE_MESSAGE)
       .reportIfAbsent(MISSING_CERTIFICATE_MESSAGE));
     register("Microsoft.App/containerApps", CertificateBasedAuthenticationCheck::checkContainerApps);
     register("Microsoft.ContainerRegistry/registries/tokens", CertificateBasedAuthenticationCheck::checkRegistriesTokens);
     register("Microsoft.DataFactory/factories/linkedservices", CertificateBasedAuthenticationCheck::checkLinkedServices);
+    register("Microsoft.SignalRService/signalR", CertificateBasedAuthenticationCheck::checkSignalRService);
+    register("Microsoft.SignalRService/webPubSub", CertificateBasedAuthenticationCheck::checkSignalRService);
```

Comment:
```suggestion
    register(List.of("Microsoft.SignalRService/signalR", "Microsoft.SignalRService/webPubSub"), CertificateBasedAuthenticationCheck::checkSignalRService);
```
Isn't this already supported?